### DISCUSSION TITLE
Does rsyncd support hot reload?

### DISCUSSION THREAD

**Discussion by qqliuxiaoran** [RsyncProject/rsync#343] [Q&A] [answered]
Hello！
Can I add/remove module or change module PARAMETERS without daemon stop？

**Comment by WayneD** [comment, marked as answer]
The [manpage](https://download.samba.org/pub/rsync/rsyncd.conf.5#LAUNCHING_THE_RSYNC_DAEMON) contains this:

> Note that you should not send the rsync daemon a HUP signal to force it to reread the rsyncd.conf file. The file is re-read on each client connection.

**Reply by qqliuxiaoran** [reply]
The file is re-read on each client connection——got it! thank you for your reply!
